wagtail_purge/wagtail_hooks.py

```diff
@@ -22,4 +19,4 @@ def register_purge_menu_item():
         reverse("purge"),
         classnames="icon icon-collapse-down",
         order=1000,
-    )
\ No newline at end of file
+    )
```

Comment:
**nitpick:** Restore the newline to the end of this file, to follow coding standards consistently